recipes/default.rb

```diff
@@ -19,7 +19,10 @@
 
 include_recipe 'ark'
 
-user node['strongdm']['user']
+user node['strongdm']['user'] do
+  home '/opt/strongdm'
+  manage_home true
```

Comment:
You may want to explicitly manage the homedir, as this will only create the dir when the user is created, but won't recreate it or adjust permissions later.

Reply:
This is pretty standard fare, so I think it's fine to leave. I've made a mental note to check on this.